src/utils/authenticateUser.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+const API_URL_PROD = 'https://api.example.com';
+
+export const authenticateUser = async (
+  username: string,
+  password: string
+): Promise<string> => {
+  const response = await axios.post(`${API_URL_PROD}/token`, {
```

Comment:
Axios is set up with a default URL, so you should only specify the path, i.e. `/token` in this case.